### DISCUSSION TITLE
instantaneous hazard

### DISCUSSION THREAD

**Discussion by weihevt** [CamDavidsonPilon/lifelines#1509] [Q&A] [answered]
Hello, is there a function which returns the instantaneous hazard rate instead of the cumulative hazard rate?

Thanks.

**Comment by CamDavidsonPilon** [comment, marked as answer]
Sure, you can use something like: https://lifelines.readthedocs.io/en/latest/Survival%20analysis%20with%20lifelines.html#smoothing-the-hazard-function

For parametric models, there is the `hazard_at_times` function: https://lifelines.readthedocs.io/en/latest/Survival%20analysis%20with%20lifelines.html#other-parametric-models-exponential-log-logistic-log-normal-and-splines

**Reply by weihevt** [reply]
Got it. Thanks so much. Actually I am not looking for the cumulative hazard rate but the simple hazard for the exponential model. And I found out some people asked this before. I can get the hazard (decay rate) for the exponential model [here](https://lifelines.readthedocs.io/en/latest/jupyter_notebooks/Piecewise%20Exponential%20Models%20and%20Creating%20Custom%20Models.html#The-Exponential-model).
